core-kotlin-modules/core-kotlin-strings-3/src/test/kotlin/com/baeldung/stringarray/StringArrayUnitTest.kt

```diff
@@ -0,0 +1,77 @@
+package com.baeldung.stringarray
+
+import org.junit.jupiter.api.Assertions.assertArrayEquals
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import java.io.BufferedReader
+import java.io.StringReader
+import java.util.*
+
+class StringArrayUnitTest {
```

Comment:
As per the style guide, test names should be written in BDD given when then convention. See the doc "baeldung - for authors - 4 - implementation code"

Reply:
Note: I'm just checking on this one with the editorial team, there is a bit of ambiguity in the style guide at the moment on this point for Kotlin, I'll update this comment when we have decided on what the style guide is here